### ISSUE TITLE
[Low Priority Bug]: Cascade particle systems which utilize either mesh-based particles or dynamic parameters causes a crash

### ISSUE THREAD

**Issue by Bennick323** [WonderInteractive/WonderPlatform#4]
### Contact Details

Nick@immersedgames.com

### What happened?

Low Priority

Loading a particle system like in the title crashes the game. I am unsure whether this carries over to Niagara-built particle systems but it definitely happens with older Cascade ones.

### Version

1.0.2 (Default)

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

### Relevant log output

```shell
3876440525.js:1323 [2022.03.08-22.57.01:113][573]LogStreaming: Display: FlushAsyncLoading: 1 QueuedPackages, 0 AsyncPackages

3876440525.js:1798 FetchAllWait 12 objects of different sizes in parallel 149.14000000059605
3876440525.js:1798 FetchAllWait 22 objects of different sizes in parallel 219.9949999898672
Access to XMLHttpRequest at 'https://analytics-staging.tytoonline.com/api/v2/game-events' from origin 'https://dev.theimmersiveweb.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at '<URL>' from origin '<URL>' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at '<URL>' from origin '<URL>' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at '<URL>' from origin '<URL>' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at '<URL>' from origin '<URL>' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at '<URL>' from origin '<URL>' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at '<URL>' from origin '<URL>' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
3876440525.js:1323 [2022.03.08-22.57.01:544][574]LogCharacterMovement: Warning: GetSimulationTimeStep() - Max iterations 8 hit while remaining time 0.089721 > MaxSimulationTimeStep (0.050) for 'AnimalActor_2147468524', movement 'Walking'

3876440525.js:1323 [2022.03.08-22.57.01:545][574]LogCharacterMovement: Warning: GetSimulationTimeStep() - Max iterations 8 hit while remaining time 0.089752 > MaxSimulationTimeStep (0.050) for 'AnimalActor_2147468521', movement 'Walking'

3876440525.js:1323 [2022.03.08-22.57.01:545][574]LogCharacterMovement: Warning: GetSimulationTimeStep() - Max iterations 8 hit while remaining time 0.085079 > MaxSimulationTimeStep (0.050) for 'AnimalActor_2147468518', movement 'Walking'

3876440525.js:1323 [2022.03.08-22.57.01:546][574]LogCharacterMovement: Warning: GetSimulationTimeStep() - Max iterations 8 hit while remaining time 0.087277 > MaxSimulationTimeStep (0.050) for 'AnimalActor_2147468515', movement 'Walking'

3876440525.js:1323 [2022.03.08-22.57.01:546][574]LogCharacterMovement: Warning: GetSimulationTimeStep() - Max iterations 8 hit while remaining time 0.088720 > MaxSimulationTimeStep (0.050) for 'AnimalActor_2147468512', movement 'Walking'

3876440525.js:1323 [2022.03.08-22.57.01:548][574]LogCharacterMovement: Warning: GetSimulationTimeStep() - Max iterations 8 hit while remaining time 0.088929 > MaxSimulationTimeStep (0.050) for 'AnimalActor_2147468509', movement 'Walking'

Access to XMLHttpRequest at 'https://analytics-staging.tytoonline.com/api/v2/game-events' from origin 'https://dev.theimmersiveweb.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at 'https://analytics-staging.tytoonline.com/api/v2/game-events' from origin 'https://dev.theimmersiveweb.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at 'https://analytics-staging.tytoonline.com/api/v2/game-events' from origin 'https://dev.theimmersiveweb.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Access to XMLHttpRequest at 'https://analytics-staging.tytoonline.com/api/v2/game-events' from origin 'https://dev.theimmersiveweb.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
3876440525.js:1323 [2022.03.08-22.57.05:831][745]LogUObjectBase: Warning: Object is not registered

3876440525.js:1323 [2022.03.08-22.57.09:236][881]LogTemp: Warning: Spawning USwarmComponent

3876440525.js:1323 [2022.03.08-22.57.09:236][881]LogTemp: Warning: USwarmComponent spawned

3876440525.js:1323 [2022.03.08-22.57.09:237][881]LogStreaming: Display: FlushAsyncLoading: 1 QueuedPackages, 0 AsyncPackages

3876440525.js:1798 FetchAllWait 10 objects of different sizes in parallel 147.39000000059605
3876440525.js:1323 [2022.03.08-22.57.09:392][881]LogTemp: Warning: Loading particle Moth, moth_vfx

3876440525.js:1798 FetchAllWait 6 objects of different sizes in parallel 81.55999998748302
3876440525.js:1323 [2022.03.08-22.57.09:477][881]LogTemp: Warning: Finished loading particle Moth, moth_vfx

Access to XMLHttpRequest at 'https://analytics-staging.tytoonline.com/api/v2/game-events' from origin 'https://dev.theimmersiveweb.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
3876440525.js:1323 [2022.03.08-22.57.09:525][882]LogRHI: Error: Failed to compile shader. Compile log:
ERROR: 0:59: 'transpose' : no matching overloaded function found
ERROR: 0:59: 'transpose' : no matching overloaded function found
ERROR: 0:59: 'transpose' : no matching overloaded function found
ERROR: 0:59: 'transpose' : no matching overloaded function found


3876440525.js:1323 [2022.03.08-22.57.09:526][882]LogRHI: Error: Failed to compile shader. Compile log:
ERROR: 0:71: 'transpose' : no matching overloaded function found
ERROR: 0:71: 'transpose' : no matching overloaded function found
ERROR: 0:71: 'transpose' : no matching overloaded function found
ERROR: 0:71: 'transpose' : no matching overloaded function found
ERROR: 0:112: 'transpose' : no matching overloaded function found
ERROR: 0:112: '=' : dimension mismatch
ERROR: 0:112: 'assign' : cannot convert from 'const mediump float' to 'highp 4-component vector of float'
ERROR: 0:113: 'transpose' : no matching overloaded function found
ERROR: 0:113: '=' : dimension mismatch
ERROR: 0:113: 'assign' : cannot convert from 'const mediump float' to 'highp 4-component vector of float'
ERROR: 0:114: 'transpose' : no matching overloaded function found
ERROR: 0:114: '=' : dimension mismatch
ERROR: 0:114: 'assign' : cannot convert from 'const mediump float' to 'highp 4-component vector of float'


3876440525.js:1323 [2022.03.08-22.57.09:529][882]LogRHI: Error: Failed to link program. Current total programs: 65 program binary bytes, last gl error 0x502, drvalloc 0
  log:
Vertex shader is not compiled.


3876440525.js:1323 [2022.03.08-22.57.09:529][882]LogOutputDevice: Warning: 

Script Stack (0 frames):


3876440525.js:1323 Fatal error: [File:C:/Users/Nick/UnrealEngine/WonderPlatformEngine/Engine/Source/Runtime/OpenGLDrv/Private/OpenGLShaders.cpp] [Line: 3463] 
Failed to link program [Program V_824DEA187C7321D1F4314954627EB52A7A33A5AE_P_EAF5A479A34E9F98998173A0F35F6E9CC0ED6656]. Current total programs: 65, precompile: 0

3876440525.js:1326 PLATFORM_BREAK
3876440525.js:1323 
appError called: 

3876440525.js:1323 Fatal error: [File:C:/Users/Nick/UnrealEngine/WonderPlatformEngine/Engine/Source/Runtime/OpenGLDrv/Private/OpenGLShaders.cpp] [Line: 3463] 
Failed to link program [Program V_824DEA187C7321D1F4314954627EB52A7A33A5AE_P_EAF5A479A34E9F98998173A0F35F6E9CC0ED6656]. Current total programs: 65, precompile: 0



3876440525.js:1323 

3876440525.js:1323 [2022.03.08-22.57.09:530][882]LogHAL: Error: appError called: Fatal error: [File:C:/Users/Nick/UnrealEngine/WonderPlatformEngine/Engine/Source/Runtime/OpenGLDrv/Private/OpenGLShaders.cpp] [Line: 3463] 
Failed to link program [Program V_824DEA187C7321D1F4314954627EB52A7A33A5AE_P_EAF5A479A34E9F98998173A0F35F6E9CC0ED6656]. Current total programs: 65, precompile: 0




3876440525.js:1326 PLATFORM_BREAK
3876440525.js:1323 [2022.03.08-22.57.09:530][882]LogHAL: === Critical error: ===




3876440525.js:1323 [2022.03.08-22.57.09:530][882]LogHAL: Fatal error: [File:C:/Users/Nick/UnrealEngine/WonderPlatformEngine/Engine/Source/Runtime/OpenGLDrv/Private/OpenGLShaders.cpp] [Line: 3463] 
Failed to link program [Program V_824DEA187C7321D1F4314954627EB52A7A33A5AE_P_EAF5A479A34E9F98998173A0F35F6E9CC0ED6656]. Current total programs: 65, precompile: 0






3876440525.js:1323 

Exiting due to error

3876440525.js:1323 [2022.03.08-22.57.09:531][882]LogExit: Executing StaticShutdownAfterError

3876440525.js:1323 [2022.03.08-22.57.09:535][882]LogGenericPlatformMisc: FPlatformMisc::RequestExit(1)

3876440525.js:1165 Aborted(native code called abort())
abort @ 3876440525.js:1165
_abort @ 3876440525.js:7122
$FGenericPlatformMisc::RequestExit(bool) @ 3961840398.wasm:0x1919f8d
$FOutputDeviceAnsiError::Serialize(char16_t const*, ELogVerbosity::Type, FName const&) @ 3961840398.wasm:0x1a8b930
$FOutputDevice::LogfImpl(char16_t const*, ...) @ 3961840398.wasm:0x1a8a6e8
$FDebug::ProcessFatalError() @ 3961840398.wasm:0x1a4091a
$auto FOpenGLDynamicRHI::RHICreateBoundShaderState_OnThisThread(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool)::$_130::operator()<FLogCategoryLogRHI, char16_t [72], char16_t const*, unsigned int, unsigned int>(FLogCategoryLogRHI const&, char16_t const (&) [72], char16_t const* const&, unsigned int const&, unsigned int const&) const @ 3961840398.wasm:0x2629c18
$FOpenGLDynamicRHI::RHICreateBoundShaderState_OnThisThread(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool) @ 3961840398.wasm:0x262967d
$FOpenGLDynamicRHI::RHICreateBoundShaderState_internal(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool)::'lambda'()::operator()() const @ 3961840398.wasm:0x264922f
$FOpenGLDynamicRHI::RHICreateBoundShaderState_internal(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool) @ 3961840398.wasm:0x2632e72
$FOpenGLDynamicRHI::RHISetGraphicsPipelineState(FRHIGraphicsPipelineState*, bool) @ 3961840398.wasm:0x2637521
$non-virtual thunk to FOpenGLDynamicRHI::RHISetGraphicsPipelineState(FRHIGraphicsPipelineState*, bool) @ 3961840398.wasm:0x26377f3
$SetGraphicsPipelineState(FRHICommandList&, FGraphicsPipelineStateInitializer const&, EApplyRendertargetOption, bool) @ 3961840398.wasm:0x26b1231
$FMeshDrawCommand::SubmitDraw(FMeshDrawCommand const&, Experimental::TRobinHoodHashSet<FGraphicsMinimalPipelineStateInitializer, DefaultKeyFuncs<FGraphicsMinimalPipelineStateInitializer, false>, TSizedDefaultAllocator<32> > const&, FRHIVertexBuffer*, int, unsigned int, FRHICommandList&, FMeshDrawCommandStateCache&) @ 3961840398.wasm:0x2c7a2d1
$SubmitMeshDrawCommandsRange(TArray<FVisibleMeshDrawCommand, TMemStackAllocator<0u> > const&, Experimental::TRobinHoodHashSet<FGraphicsMinimalPipelineStateInitializer, DefaultKeyFuncs<FGraphicsMinimalPipelineStateInitializer, false>, TSizedDefaultAllocator<32> > const&, FRHIVertexBuffer*, int, bool, int, int, unsigned int, FRHICommandList&) @ 3961840398.wasm:0x2c79faa
$FParallelMeshDrawCommandPass::DispatchDraw(FParallelCommandListSet*, FRHICommandList&) const @ 3961840398.wasm:0x2c79931
$FMobileSceneRenderer::RenderMobileBasePass(FRHICommandListImmediate&, TArrayView<FViewInfo const*, int>) @ 3961840398.wasm:0x2cd7fae
$FMobileSceneRenderer::RenderForward(FRHICommandListImmediate&, TArrayView<FViewInfo const*, int>) @ 3961840398.wasm:0x2d31705
$FMobileSceneRenderer::Render(FRHICommandListImmediate&) @ 3961840398.wasm:0x2d2e2a2
$RenderViewFamily_RenderThread(FRHICommandListImmediate&, FSceneRenderer*) @ 3961840398.wasm:0x30149f8
$FRendererModule::BeginRenderingViewFamily(FCanvas*, FSceneViewFamily*)::$_31::operator()(FRHICommandListImmediate&) const @ 3961840398.wasm:0x2fd468c
$FRendererModule::BeginRenderingViewFamily(FCanvas*, FSceneViewFamily*) @ 3961840398.wasm:0x2fd3689
$UGameViewportClient::Draw(FViewport*, FCanvas*) @ 3961840398.wasm:0x3f3c50b
$non-virtual thunk to UGameViewportClient::Draw(FViewport*, FCanvas*) @ 3961840398.wasm:0x3f3e3d6
$FViewport::Draw(bool) @ 3961840398.wasm:0x469ebf0
$UGameEngine::RedrawViewports(bool) @ 3961840398.wasm:0x3ee9a63
$UGameEngine::Tick(float, bool) @ 3961840398.wasm:0x3eee157
$FEngineLoop::Tick() @ 3961840398.wasm:0x166725
$HTML5_Tick() @ 3961840398.wasm:0x162c7c
$dynCall_v @ 3961840398.wasm:0x54f552f
(anonymous) @ 3876440525.js:1194
browserIterationFunc @ 3876440525.js:11267
callUserCallback @ 3876440525.js:7267
runIter @ 3876440525.js:7337
Browser_mainLoop_runner @ 3876440525.js:7242
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
Show 5 more frames
3660027386.html:1 Pthread 0x01f4db20 sent an error! https://dev.theimmersiveweb.com/fetchbin/3876440525.js:1168: Uncaught RuntimeError: Aborted(native code called abort())
printErr @ 3660027386.html:1
worker.onerror @ 3876440525.js:2474
3660027386.html:1 Uncaught [object ErrorEvent] https://dev.theimmersiveweb.com/fetchbin/3876440525.js 2475
window.onerror @ 3660027386.html:1
3876440525.js:2475 Uncaught ErrorEvent {isTrusted: true, message: 'Uncaught RuntimeError: Aborted(native code called abort())', filename: 'https://dev.theimmersiveweb.com/fetchbin/3876440525.js', lineno: 1168, colno: 10, …}
worker.onerror @ 3876440525.js:2475
3660027386.html:1 Uncaught RuntimeError: Aborted(native code called abort()) https://dev.theimmersiveweb.com/fetchbin/3876440525.js 1168
window.onerror @ 3660027386.html:1
3876440525.js:1168 Uncaught RuntimeError: Aborted(native code called abort())
    at abort (3876440525.js:1168:10)
    at _abort (3876440525.js:7122:2)
    at FGenericPlatformMisc::RequestExit(bool) (3961840398.wasm:0x1919f8d)
    at FOutputDeviceAnsiError::Serialize(char16_t const*, ELogVerbosity::Type, FName const&) (3961840398.wasm:0x1a8b930)
    at FOutputDevice::LogfImpl(char16_t const*, ...) (3961840398.wasm:0x1a8a6e8)
    at FDebug::ProcessFatalError() (3961840398.wasm:0x1a4091a)
    at auto FOpenGLDynamicRHI::RHICreateBoundShaderState_OnThisThread(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool)::$_130::operator()<FLogCategoryLogRHI, char16_t [72], char16_t const*, unsigned int, unsigned int>(FLogCategoryLogRHI const&, char16_t const (&) [72], char16_t const* const&, unsigned int const&, unsigned int const&) const (3961840398.wasm:0x2629c18)
    at FOpenGLDynamicRHI::RHICreateBoundShaderState_OnThisThread(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool) (3961840398.wasm:0x262967d)
    at FOpenGLDynamicRHI::RHICreateBoundShaderState_internal(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool)::'lambda'()::operator()() const (3961840398.wasm:0x264922f)
    at FOpenGLDynamicRHI::RHICreateBoundShaderState_internal(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool) (3961840398.wasm:0x2632e72)
abort @ 3876440525.js:1168
_abort @ 3876440525.js:7122
$FGenericPlatformMisc::RequestExit(bool) @ 3961840398.wasm:0x1919f8d
$FOutputDeviceAnsiError::Serialize(char16_t const*, ELogVerbosity::Type, FName const&) @ 3961840398.wasm:0x1a8b930
$FOutputDevice::LogfImpl(char16_t const*, ...) @ 3961840398.wasm:0x1a8a6e8
$FDebug::ProcessFatalError() @ 3961840398.wasm:0x1a4091a
$auto FOpenGLDynamicRHI::RHICreateBoundShaderState_OnThisThread(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool)::$_130::operator()<FLogCategoryLogRHI, char16_t [72], char16_t const*, unsigned int, unsigned int>(FLogCategoryLogRHI const&, char16_t const (&) [72], char16_t const* const&, unsigned int const&, unsigned int const&) const @ 3961840398.wasm:0x2629c18
$FOpenGLDynamicRHI::RHICreateBoundShaderState_OnThisThread(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool) @ 3961840398.wasm:0x262967d
$FOpenGLDynamicRHI::RHICreateBoundShaderState_internal(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool)::'lambda'()::operator()() const @ 3961840398.wasm:0x264922f
$FOpenGLDynamicRHI::RHICreateBoundShaderState_internal(FRHIVertexDeclaration*, FRHIVertexShader*, FRHIHullShader*, FRHIDomainShader*, FRHIPixelShader*, FRHIGeometryShader*, bool) @ 3961840398.wasm:0x2632e72
$FOpenGLDynamicRHI::RHISetGraphicsPipelineState(FRHIGraphicsPipelineState*, bool) @ 3961840398.wasm:0x2637521
$non-virtual thunk to FOpenGLDynamicRHI::RHISetGraphicsPipelineState(FRHIGraphicsPipelineState*, bool) @ 3961840398.wasm:0x26377f3
$SetGraphicsPipelineState(FRHICommandList&, FGraphicsPipelineStateInitializer const&, EApplyRendertargetOption, bool) @ 3961840398.wasm:0x26b1231
$FMeshDrawCommand::SubmitDraw(FMeshDrawCommand const&, Experimental::TRobinHoodHashSet<FGraphicsMinimalPipelineStateInitializer, DefaultKeyFuncs<FGraphicsMinimalPipelineStateInitializer, false>, TSizedDefaultAllocator<32> > const&, FRHIVertexBuffer*, int, unsigned int, FRHICommandList&, FMeshDrawCommandStateCache&) @ 3961840398.wasm:0x2c7a2d1
$SubmitMeshDrawCommandsRange(TArray<FVisibleMeshDrawCommand, TMemStackAllocator<0u> > const&, Experimental::TRobinHoodHashSet<FGraphicsMinimalPipelineStateInitializer, DefaultKeyFuncs<FGraphicsMinimalPipelineStateInitializer, false>, TSizedDefaultAllocator<32> > const&, FRHIVertexBuffer*, int, bool, int, int, unsigned int, FRHICommandList&) @ 3961840398.wasm:0x2c79faa
$FParallelMeshDrawCommandPass::DispatchDraw(FParallelCommandListSet*, FRHICommandList&) const @ 3961840398.wasm:0x2c79931
$FMobileSceneRenderer::RenderMobileBasePass(FRHICommandListImmediate&, TArrayView<FViewInfo const*, int>) @ 3961840398.wasm:0x2cd7fae
$FMobileSceneRenderer::RenderForward(FRHICommandListImmediate&, TArrayView<FViewInfo const*, int>) @ 3961840398.wasm:0x2d31705
$FMobileSceneRenderer::Render(FRHICommandListImmediate&) @ 3961840398.wasm:0x2d2e2a2
$RenderViewFamily_RenderThread(FRHICommandListImmediate&, FSceneRenderer*) @ 3961840398.wasm:0x30149f8
$FRendererModule::BeginRenderingViewFamily(FCanvas*, FSceneViewFamily*)::$_31::operator()(FRHICommandListImmediate&) const @ 3961840398.wasm:0x2fd468c
$FRendererModule::BeginRenderingViewFamily(FCanvas*, FSceneViewFamily*) @ 3961840398.wasm:0x2fd3689
$UGameViewportClient::Draw(FViewport*, FCanvas*) @ 3961840398.wasm:0x3f3c50b
$non-virtual thunk to UGameViewportClient::Draw(FViewport*, FCanvas*) @ 3961840398.wasm:0x3f3e3d6
$FViewport::Draw(bool) @ 3961840398.wasm:0x469ebf0
$UGameEngine::RedrawViewports(bool) @ 3961840398.wasm:0x3ee9a63
$UGameEngine::Tick(float, bool) @ 3961840398.wasm:0x3eee157
$FEngineLoop::Tick() @ 3961840398.wasm:0x166725
$HTML5_Tick() @ 3961840398.wasm:0x162c7c
$dynCall_v @ 3961840398.wasm:0x54f552f
(anonymous) @ 3876440525.js:1194
browserIterationFunc @ 3876440525.js:11267
callUserCallback @ 3876440525.js:7267
runIter @ 3876440525.js:7337
Browser_mainLoop_runner @ 3876440525.js:7242
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
requestAnimationFrame (async)
requestAnimationFrame @ 3876440525.js:7631
Browser_mainLoop_scheduler_rAF @ 3876440525.js:7144
Browser_mainLoop_runner @ 3876440525.js:7246
Show 5 more frames
SaveLog()
VM272:1 Uncaught ReferenceError: SaveLog is not defined
    at <anonymous>:1:1
(anonymous) @ VM272:1
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
